BillCode.js

```diff
@@ -5,114 +5,229 @@ function onOpen() {
     .addToUi();
 }
 
+function getPeople() {
+  const sheet = SpreadsheetApp.getActiveSpreadsheet().getSheetByName('People');
+  if (!sheet) return [];
+
+  const data = sheet.getDataRange().getValues();
+  return data.slice(1).map(row => ({ name: row[0], email: row[1] }));
+}
+
 function enterBillDetails() {
-  const html = HtmlService.createHtmlOutputFromFile('BillForm')
-    .setWidth(400)
-    .setHeight(600);
-  SpreadsheetApp.getUi().showModalDialog(html, 'Enter Bill Details');
+  const people = getPeople(); // Returns an array of objects: [{ name: "John Doe", email: "john@example.com" }, ...]
+  const html = HtmlService.createTemplateFromFile('BillForm');
+  html.people = JSON.stringify(people); // Properly encode as JSON string
+  SpreadsheetApp.getUi().showModalDialog(
+    html.evaluate().setWidth(400).setHeight(600),
+    'Enter Bill Details'
+  );
 }
 
 function addBillToSheet(data) {
   const sheet = SpreadsheetApp.getActiveSpreadsheet().getActiveSheet();
-  
-  // Check if sheet is empty and add headers if needed
+
   if (sheet.getLastRow() === 0) {
-    const headers = [
-      'Unique ID', 
-      'Description', 
-      'Date', 
-      'Total Amount', 
-      'Who Paid', 
-      'Contribution Split', 
-      'Balance Split', 
-      'Entry Folder Link'
-    ];
-    
-    sheet.appendRow(headers);
-    
-    // Style headers: bold text and pastel background
-    const headerRange = sheet.getRange(1, 1, 1, headers.length);
-    headerRange.setFontWeight('bold');
-    headerRange.setBackground('#E5E5FA'); // Light pastel lavender color
+    // Add headers if the sheet is empty
+    sheet.appendRow(['Unique ID', 'Description', 'Date', 'Total Amount', 'Who Paid', 'Contribution Split', 'Balance Split', 'Folder Link']);
   }
-  
-  // Set the unique ID to 1 for the first entry, or continue with the next number
+
   const lastRow = sheet.getLastRow();
-  const uniqueId = lastRow === 0 ? 1 : lastRow; // If no rows, set to 1; otherwise, just use the row number.
+  const uniqueId = lastRow === 0 ? 1 : lastRow + 1;
```

Comment:
Get rid of the + 1, it makes the first entry have a Unique ID of 2 instead of 1

Reply:
Done in `[fix Removed +1 Entry](https://github.com/PiqueDevelopment/Evenly--Bill-Expense-Application/pull/16/commits/3e51090c3db734b72b51697e5ec247579231552d)`